.readthedocs.yaml

```diff
@@ -0,0 +1,25 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.12"
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+   configuration: docs/conf.py
+
+# Declare the Python requirements required to build your docs.
+# Here we use the optional dependencies required in the [docs] group.
+python:
+  install:
+    - method: pip
+      path: .
+      extra_requirements:
+        - docs
```

Comment:
Do I understand this correctly that it will use the requirements in the [docs] group of pyproject.toml?

Reply:
That's right, it's just a way to avoid repeating our list of documentation dependencies.

Reply:
Nice